orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -31,6 +32,12 @@ def __init__(self, state, output_info):
         self.state = state
         self.output_info = output_info
 
+@dataclass
+class SkipCheck():
+    skip_build_gradient_graph: bool = False
+    skip_create_execution_agent: bool = False
+    skip_device_check: bool = False
+
```

Comment:
`self._fast_path` can be replaced by

```suggestion
    def is_disabled(self):
       return not self.skip_build_gradient_graph and not self.skip_create_execution_agent and not self.skip_device_check
```

In the next comment I also proposed another alternative

Reply:
refactored with IntFlag